spec/system/service_unavailable_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.describe 'Service Unavailable' do
+  before do
+    allow(ENV).to receive(:[]).and_call_original
+    allow(ENV).to receive(:[]).with('MAINTENANCE').and_return('true')
+  end
+
+  context 'when the service is unavailable and user navigates to home' do
+    before do
+      visit '/service-unavailable'
```

Comment:
Is this line doing anything?

Reply:
[37aa9b6](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/941/commits/37aa9b6ee52227bd8b6d9dd34c555eab44b37802)